winit/src/window.rs

```diff
@@ -155,27 +155,85 @@ impl View for Window {
                 }
             }
 
+            WindowEvent::SetTitle(title) => {
+                self.window().set_title(title);
+            }
+
+            WindowEvent::SetSize(size) => {
+                self.window().set_inner_size(LogicalSize::new(size.width, size.height));
+            }
+
+            WindowEvent::SetMinSize(size) => {
+                self.window().set_min_inner_size(Some(LogicalSize::new(size.width, size.height)));
```

Comment:
You are not able to remove the `min_inner_size` currently right? I think it would make more sense to pass an `Option` here instead.

Reply:
Ah yes, good point!